### PR TITLE
Azfar fixes2

### PR DESCRIPTION
**Pull request by cliffckerr**

### CODE REVIEW

**Comment by cliffckerr** [comment]
@robynstuart please review, simple changes, adds keys() method to parameters and fixes results to include data for popsize

File: optima/model.py [hunk 1]
```diff
@@ -782,8 +781,8 @@ def model(simpars=None, settings=None, verbose=None, benchmark=False, die=True):
                     change[tx[cd4],:]  = dT[cd4]
             people[:,:,t+1] = people[:,:,t] + change # Update people array
             newpeople = popsize[:,t+1]-people[:,:,t+1].sum(axis=0) # Number of people to add according to simpars['popsize'] (can be negative)
-            people[susreg,pop,t+1] += newpeople[pop] - numcirc[pop,t] # Add new people, then subtract if from circumcision
-            people[circ,pop,t+1]   += numcirc[pop,t] # And add these people into the circumcised compartment
+            people[susreg,:,t+1] += newpeople - numcirc[pop,t] # Add new people, then subtract if from circumcision
```
Comment (robynstuart) [inline optima/model.py]: are you sure about this? the previous way looks more correct...

Comment (cliffckerr) [inline optima/model.py]: it's not in a population loop -- it used to be, but it's not any mo -- AUGH, still a bug

File: optima/parameters.py [hunk 2]
```diff
@@ -646,6 +666,10 @@ def __init__(self, p=None, m=1, start=2000, **defaultargs):
         self.m = m # Multiplicative metaparameter, e.g. 1
         self.start = start # Year for which population growth start is calibrated to
     
+    def keys(self):
+        ''' Return the valid keys for using with this parameter '''
+        return self.p.keys()
+
```
Comment (robynstuart) [inline optima/parameters.py]: this appears to be in here twice... ?

Comment (robynstuart) [inline optima/parameters.py]: oh no i see

**Comment by cliffckerr** [comment]
subsumed into dev-plotbocs

**Pull request closed without merging**